simulation/traffic_simulator/include/traffic_simulator/data_type/follow_trajectory.hpp

```diff
@@ -52,9 +52,21 @@ struct Parameter
   bool dynamic_constraints_ignorable;
 
   /*
-     If Parameter::closed is true, this variable is always assigned false.
+     This data member corresponds to the attribute
+     TimeReference.Timing.domainAbsoluteRelative of ASAM OpenSCENARIO 1.2. the
+     OpenSCENARIO standard does not define the behavior when the value of
+     domainAbsoluteRelative is "relative". The standard only states "Definition
+     of time value context as either absolute or relative", and it is
+     completely unclear when the relative time starts.
+
+     This implementation has interpreted the specification as follows: Relative
+     time starts from the start of FollowTrajectoryAction or from the time of
+     reaching the previous "waypoint with arrival time".
+
+     Note: If Parameter::closed is true, this variable is always assigned
+     false.
   */
-  bool timing_is_absolute;
+  bool time_specification_of_trajectory_is_absolute_simulation_time;
```

Comment:
The word `time specification` is really exist?

Reply:
exists.

Reply:
What does "The word time specification is really exist?" mean?

Reply:
[7d1284c](https://github.com/tier4/scenario_simulator_v2/pull/906/commits/7d1284cbe4542bb045907cd2d6d94a3dc8de6aaf) fixes this.